optuna/trial.py

```diff
@@ -559,6 +560,16 @@ def system_attrs(self):
 
         return self.storage.get_trial_system_attrs(self._trial_id)
 
+    @property
+    def datetime_start(self):
+        # type: () -> Optional[datetime]
+        """Return datetime of the trial started.
```

Comment:
I think that this doc can be rewritten to something simpler as follows: 
```suggestion
        """Return start datetime.
```